src/appendix/glossary.md

```diff
@@ -75,14 +75,14 @@ Term                                                  | Meaning
 <span id="substs">substs</span>                &nbsp; |  The substitutions for a given generic type or item (e.g. the `i32`, `u32` in `HashMap<i32, u32>`).
 <span id="sysroot">sysroot</span>              &nbsp; |  The directory for build artifacts that are loaded by the compiler at runtime. ([see more](../building/bootstrapping.html#what-is-a-sysroot))
 <span id="tag">tag</span>                      &nbsp; |  The "tag" of an enum/generator encodes the [discriminant](#discriminant) of the active variant/state.  Tags can either be "direct" (simply storing the discriminant in a field) or use a ["niche"](#niche).
-<span id="tcx">tcx</span>                      &nbsp; |  The "typing context" (`TyCtxt`), main data structure of the compiler. ([see more](../ty.md))
+<span id="tcx">`tcx`</span>                    &nbsp; |  Conventional variable name for the "typing context" (`TyCtxt`), main data structure of the compiler. ([see more](../ty.md))
```

Comment:
Maybe too much of a nitpick, but what about "standard"?

Reply:
Maybe "Standard name" instead of "variable name"? (It's a common field name as well...)

Reply:
"Standard name" is confusing because it sounds like it means the name of the type too (which is in fact `TyCtxt`). I think "variable name" can cover field name as well. After all, usually it's not a **vari**able, but an immutable binding anyway ;)